README.md

```diff
@@ -41,7 +41,7 @@ Pkg.add("CPLEX")
 ## Help! I got `LoadError: Unable to locate CPLEX installation`
 
 Which version of CPLEX are you trying to install? Currently, CPLEX.jl only
-supports 1280 and 1290 given recent changes to
+supports 1280, 1290, and 12100 given recent changes to
```

Comment:
Oh, by the way, maybe write 12.8 (or 12.8.0, if you prefer)? That should be easier to read.

Reply:
Sure :)